#      0 ------- 1
#      |         |
#      |         |
#      |         |
#      3 ------- 2
yolov7_ros/Keypoint2Di[4] corners
